Rakefile

```diff
@@ -155,6 +156,17 @@ namespace :docs do
   end
 end
 
+namespace :container do
+  desc "(re)build docker container for tests"
+  rototiller_task :build do |command|
+    command.name = "docker build ./ --file Dockerfile-tests --tag pcr-internal.puppet.net/slv/rototiller:latest"
+  end
+  desc "push docker container for tests to Puppet Container Registry. You need to be logged into it first `docker login -u TOKEN --password <token from pcr> pcr-internal.puppet.net`"
+  rototiller_task :push do |command|
+    command.name =  "docker push pcr-internal.puppet.net/slv/rototiller:latest"
```

Comment:
the pcr uri should probably be in a varaible near the top or something, since it is used more than once, and highly specific.